core/api/src/services/mongoose/schema.ts

```diff
@@ -307,6 +303,26 @@ AccountSchema.index({
 
 export const Account = mongoose.model<AccountRecord>("Account", AccountSchema)
 
+const QuizSchema = new Schema<QuizRecord>({
+  accountId: {
```

Comment:
maybe because it is the first PR but having a key only based on accountId will allow the delete->claim rewards> create cycle

Reply:
this is not changing the behavior compared to before? the goal of this PR is to create a time bound earn / section. other way to prevent yields for people re-creating account can be part of a different PR